File: xskeleton-lib/src/main/java/com/mcgrady/xskeleton/http/interf/ResponseErrorListener.java
package com.mcgrady.xskeleton.http.interf;

import android.content.Context;

/**
 * Created by mcgrady on 2019/4/26.
 */
public interface ResponseErrorListener {

    void handlerResponseError(Context context, Throwable t);

    ResponseErrorListener EMPTY = (context, t) -> { };
}

File: xskeleton-lib/src/main/java/com/mcgrady/xskeleton/http/interf/ResponseErrorListener.kt
package com.mcgrady.xskeleton.http.interf

import android.content.Context

/**
 * Created by mcgrady on 2019/4/26.
 */
interface ResponseErrorListener {
    fun handlerErrorResultCode(context: Context?, resultCode: Int, errMsg: String?): Boolean
    fun handlerResponseError(context: Context?, t: Throwable?) //ResponseErrorListener EMPTY = (context, t) -> {};
}